dss/dynamodb/__init__.py

```diff
@@ -3,53 +3,54 @@
 from dss.util.aws.clients import dynamodb as db  # type: ignore
 
 
-def put_item(table: str, value: str, hash_key: str, sort_key: str):
+def format_item(value, hash_key, sort_key):
+    item = {'hash_key': {'S': hash_key}}
+    if value:
+        item['body'] = {'S': value}
+    if sort_key:
+        item['sort_key'] = {'S': sort_key}
+    return item
+
+
+def put_item(table: str, value: str, hash_key: str, sort_key: str=None, overwrite: str=None):
```

Comment:
Is it safe to pass in `None` for `value`? Ideally `value` would come after `hash_key` and `sort_key`.

Reply:
Will test.

Reply:
It can't pass value as `None` anymore unless explicitly done.  It couldn't before either, but this makes it clearer.